### PR TITLE
Issue #164/165/168: Update UG/DG for sort, find and view 

### LINKED ISSUE #164: Update `find` command in UG/DG
Updates to be made to (but not limited to) user stories, use cases, implementation, features sections.

### LINKED ISSUE #165: Update `sort` command in UG/DG
Updates to be made to (but not limited to) user stories, use cases, implementation, features sections.

### LINKED ISSUE #168: Update `view` command in UG/DG
Updates to be made to (but not limited to) user stories, use cases, implementation, features sections.

Tag teaming here, let's update each other on progress and see what we can get done! :)

### LINKED ISSUE #217: Change UI of `Candidate List` to follow `Interview List`

### PR DESCRIPTION
**Pull request by charmainehly**
Still a work in progress. 

**Currently PR includes:**
- (UG) additions of the above commands to the features section of UG
- (UG) midpoint update of gui template
- (GUI) minor edits to gui to standardise the colour scheme and style
- (Functionality) shortening of attribute fields used in the sort and find command to standardise with other commands
- (Functionality) fixing bug for find command for availability (to take out the `availability: ` string in the validity check)
- (EDIT) Shifted priority of `sort` command in user stories from `low` to `medium`
- (WIP) Implementation of `sort` and `view` commands
- (WIP) Refactoring of TAs to candidates

<br> **Following points up for discussion!!! (to delete, modify etc.!!)**
- (UG) reorganisation of command summary for user's ease of understanding
- (UG) addition of a section on how to navigate the display (i.e. our three panels)

<br> Closes #168 , Closes #164 , Closes #165 , Closes #217  - will open issues for finalising of DG and WIP sections!

### CODE REVIEW

File: docs/UserGuide.md
Comment (charmainehly) [inline docs/UserGuide.md:7]: Rationale of change - to improve phrasing and emphasise on core functionality/value proposition being the scheduling functionality. Qualifications of candidates is out of scope.

Adjusted phrasing to make subject of sentence clearer.

**Comment by charmainehly** [comment]
@domlimm @LeeEnEn good to go for merging once yall have reviewed! - i'll be opening new issues for outstanding tasks on the DG so as to update progress! The failing `codecov` is attributed to the edits to code lines, with no additional test cases added.

**Comment by domlimm** [comment]
> @domlimm @LeeEnEn good to go for merging once yall have reviewed! - i'll be opening new issues for outstanding tasks on the DG so as to update progress! The failing `codecov` is attributed to the edits to code lines, with no additional test cases added.

@charmainehly Noted! Was waiting for your go to review and merge after seeing you assigned us but stated that there were more things to add. I'll review right now! 👍🏻

File: docs/DeveloperGuide.md
Comment (domlimm) [inline docs/DeveloperGuide.md:379]: Would we need this?

```suggestion
2. `course` (accepted user input abbreviated as `c`)
```

File: docs/UserGuide.md
Comment (domlimm) [reply docs/UserGuide.md:7]: Agreed!

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java
Comment (domlimm) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java:55]: Love that you changed this to ensure it's consistent! 😄 👍🏻

File: src/main/java/seedu/address/model/candidate/predicate/CandidateContainsKeywordsPredicate.java
Comment (domlimm) [inline src/main/java/seedu/address/model/candidate/predicate/CandidateContainsKeywordsPredicate.java:35]: Nice refactor!

File: src/main/resources/view/DarkTheme.css
Comment (domlimm) [inline src/main/resources/view/DarkTheme.css:496]: The bracket `}` at the end should probably be on the next line right? 🙈

**Review by domlimm — CHANGES_REQUESTED** [review]
@charmainehly  Thanks for refactoring `TA/s` to `candidate/s`! All LGTM! Good job! 💯 

Was thinking for the `docs/images/Ui.png`, we could add a more in depth image showing:
1. Some interviews scheduled
2. Focused on a candidate
3. Show some command typed in with the feedback filled

File: src/main/resources/view/DarkTheme.css
Comment (charmainehly) [reply src/main/resources/view/DarkTheme.css:496]: will make the change for this!

File: docs/DeveloperGuide.md
Comment (charmainehly) [reply docs/DeveloperGuide.md:379]: not relevant here! Have kept the user input for find command as `course`! will update if we deem it necessary to shorten `course` to `c` as well?

**Comment by charmainehly** [comment]
Thanks @domlimm have addressed the comments. Agreed with the change to Ui.png, but lets finalise it at the end? :)

File: docs/DeveloperGuide.md
Comment (domlimm) [reply docs/DeveloperGuide.md:379]: For `add` and `edit`, both are using `c` char

**Review by domlimm — APPROVED** [review]
Thanks for making the change! LGTM! 👍🏻

File: docs/DeveloperGuide.md
Comment (charmainehly) [reply docs/DeveloperGuide.md:379]: this section is for the `find` command! not the `edit` and `add`! Hmmm, unless your suggestion is to change `course` to `c`, `phone` to `p` and `email` to `e` as well for the `find` command?

File: docs/UserGuide.md
Comment (LeeEnEn) [reply docs/UserGuide.md:7]: Yep, I agree too!

File: docs/UserGuide.md
Comment (LeeEnEn) [inline docs/UserGuide.md:42]: Perhaps the `"jane"` should be `alex` instead? 🤔

Comment (charmainehly) [reply docs/UserGuide.md:42]: oh yah cos loading from the sample data right! alright lemme make that change!! :-) thank you!!!

File: docs/DeveloperGuide.md
Comment (domlimm) [reply docs/DeveloperGuide.md:379]: hey @charmainehly, yup i know it is for that. i'm okay either ways. i just thought of the consistency between all the commands that's all. tbh, if no t ime, let's let it go. 👍🏻 i'll let you decide boss!

**Pull request merged**
